conf/pulsar_env.sh

```diff
@@ -67,7 +67,7 @@ done
 # Garbage collection options
 if [ -z "$PULSAR_GC" ]; then
   PULSAR_GC="-XX:+PerfDisableSharedMem -XX:+AlwaysPreTouch"
-  if [[ $JAVA_MAJOR_VERSION -ge 21 ]]; then
```

Comment:
Why not keep "ge" here and other places?

Reply:
https://openjdk.org/jeps/474 , generational is default in 23 and non-generational has been removed in 24